### PR TITLE
Fire TV Events to Amplitude

### PR DESCRIPTION
**Pull request by fbertsch**
This follows the format of previous new pings. I kept device and arch optional, since they are included in the schema.

### CODE REVIEW

**Comment by fbertsch** [comment]
Tested on dev, here is an example chart: https://analytics.amplitude.com/mozilla-corp/chart/6ab6nnu

File: configs/fire_tv_events_schemas.json [hunk 1]
```diff
@@ -0,0 +1,1545 @@
+{
+    "source": "telemetry",
+    "filters": {
+        "appName": [
+            "FirefoxForFireTV", 
+            "OTHER"
```
Comment (akkomar) [inline configs/fire_tv_events_schemas.json]: Is "OTHER" needed?

File: configs/fire_tv_events_schemas.json [hunk 2]
```diff
@@ -0,0 +1,1545 @@
...
+                        "properties": {
+                            "category": {
+                                "enum": [
+                                    "action"
+                                ], 
+                                "type": "string"
+                            }, 
+                            "method": {
+                                "enum": [
+                                    "click_or_voice"
```
Comment (akkomar) [inline configs/fire_tv_events_schemas.json]: That's not fixable at this level, but I'm wondering if they'd want to know which one was that (click or voice).

File: configs/fire_tv_events_schemas.json [hunk 3]
```diff
@@ -0,0 +1,1545 @@
...
+                                ], 
+                                "type": "string"
+                            }, 
+                            "timestamp": {
+                                "minimum": 0, 
+                                "type": "number"
+                            }, 
+                            "value": {
+                                "enum": [
+                                    "play"
```
Comment (akkomar) [inline configs/fire_tv_events_schemas.json]: Typo? It's `"media - pause"` event.

```suggestion
                                    "pause"
```

File: configs/fire_tv_events_schemas.json [hunk 4]
```diff
@@ -0,0 +1,1545 @@
...
+                                ], 
+                                "type": "string"
+                            }, 
+                            "timestamp": {
+                                "minimum": 0, 
+                                "type": "number"
+                            }, 
+                            "value": {
+                                "enum": [
+                                    "pause"
```
Comment (akkomar) [inline configs/fire_tv_events_schemas.json]: Same as above?
```suggestion
                                    "play"
```

File: configs/fire_tv_events_schemas.json (hunk 1 above)
Comment (fbertsch) [reply configs/fire_tv_events_schemas.json]: Unfortunately it is, since we never moved `FirefoxForFireTV` out of the `OTHER` bucket

Comment (fbertsch) [reply configs/fire_tv_events_schemas.json]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1553585. Once we backfill and that is completed we can remove `OTHER`.

File: configs/fire_tv_events_schemas.json (hunk 2 above)
Comment (fbertsch) [reply configs/fire_tv_events_schemas.json]: Good question. Might be worth bringing to Ashley's attention.

File: configs/fire_tv_events_schemas.json (hunk 3 above)
Comment (fbertsch) [reply configs/fire_tv_events_schemas.json]: Good catch!

**Pull request merged**